recipe/meta.yaml

```diff
@@ -17,7 +17,10 @@ build:
 
 requirements:
   build:
-    - svn
+    # something is broken in the latest svn package
+    - svn 1.9.4
+    # we need to pin expat to get svn 1.9.4 to work
+    - expat 2.2.1
```

Comment:
@leouieda it seems that the latest `svn` is broken (or maybe a bad mix with `conda-forge` deps?).
Pinning it to the previous version, 1.9.4, and pinning `expat` to `2.2.1` seems to solve it.

Reply:
Yes, we may have inadequate pinning in our svn recipe.

Reply:
But that does not explain the failure with `svn 1.9.7`, right?

Reply:
It could - if 1.9.7 is getting installed with some mix of defaults and conda-forge packages that are incompatible

svn itself looks fine, but it might be one of the transitive deps causing problems.

```
depends |  
-- | --
0 | "apr >=1.6.3,<2.0a0"
1 | "libgcc-ng 7.2.0.*"
2 | "serf >=1.3.9,<1.4.0a0"
3 | "sqlite >=3.20.1,<4.0a0"
4 | "zlib >=1.2.11,<1.3.0a0"
```

Reply:
APR:


depends |  
-- | --
0 | "expat >=2.2.5,<3.0a0"
1 | "libgcc-ng 7.2.0.*"

serf:


  | "apr >=1.6.3,<2.0a0"
-- | --
1 | "expat >=2.2.5,<3.0a0"
2 | "libgcc-ng 7.2.0.*"
3 | "openssl >=1.0.2n,<1.0.3a"
4 | "zlib >=1.2.11,<1.3.0a0"

Reply:
Got it. Thanks for helping is debug this.

@leouieda regardless of the `svn` issue here it would be nice to have `gmt`'s source in a more modern form :smile:

Reply:
Thanks for looking into this @ocefpaf and @msarahan. 

> it would be nice to have gmt's source in a more modern form

We all agree on that. I'm slowly trying to get them on Github but it takes a time and effort to learn a new tool. We'll get there some time this year for sure.

Reply:
FWIW, github has an svn interface.  They can keep svn as their tool, but use github as their repo, which is either the best of both worlds, or a horrible headache.  I'm not sure which - haven't used it in earnest.

Reply:
Yeah, I'm hoping to get them onboard with that and slowly get them out of svn. It helps that one of the devs is already on Github. Don't want to risk the horrible headache too much